docusaurus.config.ts

```diff
@@ -35,6 +35,10 @@ const config: Config = {
         theme: {
           customCss: './src/css/custom.css',
         },
+        gtag: {
```

Comment:
I'm not very familiar with this, but just wanted to confirm: on the [docusarus site](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-gtag#installation), it says you can add this directly if you're using the `@docusaurus/preset-classic` preset, but in our config we are using only 'classic', not sure if it's the same.

This is the example they gave:
`module.exports = {
  presets: [
    [
      '@docusaurus/preset-classic',
      {
        gtag: {
          trackingID: 'G-999X9XX9XX',
          anonymizeIP: true,
        },
      },
    ],
  ],
};`

and on ours, we're not using `@docusaurus/preset-classic` but `classic` directly. Have you tested if the tag is firing?

Reply:
can we figure this out quickly and get merged as soon as possible?  the site has been shared widely and I want analytics. @yurikoinaba

Reply:
It's the same thing